README.md

```diff
@@ -34,7 +28,7 @@ Internet browser: Firefox, Google Chrome, Opera...
 Download the code or copy the git repository to your computer.
 
 ### Usage
-Run the index.html file.
+Run the bubble_sort.rb file in your terminal.
 
 ### Run tests
 Linters errors
```

Comment:
- [ ] This title is not meaningful and doesn't contain any instructions to test your project. Please remove it or if you want to give instructions about the linter checks, please explain them explicitly.